server/game/cards/07.2-FP/DeliberateInfection.js

```diff
@@ -0,0 +1,36 @@
+const ActionCard = require('../../actioncard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class DeliberateInfection extends ActionCard {
+    setupCardAbilities(ability) {
+        this.whileAttached({
+            condition: () => this.parent.getType() === 'dude',
+            effect: [
+                ability.effects.modifyUpkeep(1),
+                ability.effects.modifyInfluence(-1)
+            ]
+        });
+        this.action({
+            title: 'Deliberate Infection',
+            playType: 'cheatin resolution',
+            condition: () => !this.parent,
+            target: {
+                choosingPlayer: 'thisIfShootout',
+                cardCondition: { location: 'play area', controller: 'opponent', participating: true },

```

Comment:
I do not think we want to set `participating: true` since this can be played during Gamblin' phase. I guess better would be to use condition and require target to be participating only if in shootout.

Reply:
It seems to work fine for coachwhip.

Reply:
AFAICT CardMatcher.js will match participating as true any time a shootout isn't happening